Client/mods/deathmatch/logic/CDeathmatchObject.cpp

```diff
@@ -96,6 +98,14 @@ void CDeathmatchObject::UpdateMovement()
 
     SetOrientation(positionRotation.m_vecPosition, positionRotation.m_vecRotation);
 
+    CLuaArguments Arguments;
+    CVector       currentPosition;
+    this->GetPosition(currentPosition);
+    Arguments.PushNumber(currentPosition.fX);
+    Arguments.PushNumber(currentPosition.fY);
+    Arguments.PushNumber(currentPosition.fZ);
+    this->CallEvent("onClientObjectMove", Arguments, true);
```

Comment:
This is going to crash the client if somebody decides to use `stopObject` or `moveObject` inside that event handler.